cfgov/v1/tests/views/test_login_views.py

```diff
@@ -63,7 +63,7 @@ def test_check_permissions_no_auth(self):
     def test_check_permissions_next_url_does_not_exist(self):
         self.client.login(username='admin', password='admin')
         response = self.client.get('/login/check_permissions/?next=/badurl/')
-        self.assertRedirects(response, '/login/welcome/')
+        self.assertRedirects(response, '/admin/')
 
     def test_check_permissions_next_permissions_problem(self):
```

Comment:
Shouldn't these two tests and the one above (that I can't select to comment on) be covered by stock Wagtail test coverage?

Reply:
Unfortunately, no, because the [`check_permissions`](https://github.com/cfpb/cfgov-refresh/blob/1b7b9b64d70753f74487bf5f848cbed9d198cc21/cfgov/v1/views/__init__.py#L129) view is something custom to cfgov-refresh (although, like our custom login view, it would be good to look at deprecating this).